builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -986,4 +986,55 @@ def retrofitMandatoryFields(Map<String, SBR_Rule> aPath2RuleMap,
   return accumulator
 }
 
+/**
+* Prepare serverless.yml from
+* config
+**/
+def prepareServerlessYml(aConfig, env) {
+	def deploymentDescriptor = aConfig['deployment_descriptor']
+	try {
+		def appContent = readFile('application.yml').trim()
+		if(!appContent.isEmpty()) deploymentDescriptor = appContent
+		} catch(e) {
+			echo "Error occured while reading application.yml. The default value from config will be used. Exception is $e"
+		}
+			
+		def doc = deploymentDescriptor  ? readYaml(text: deploymentDescriptor ) : [:] // If no descriptor present then simply making an empty one. The readYaml default behavior is to return empty string back that is harful as Map not String is expected below
+		
+		context =["environment_logical_id": env,
```

Comment:
configLoader is missing over here. better to frame the context in the parent function pass it here